README.md

```diff
@@ -166,6 +166,11 @@ assert('http://example.com/soap/service' == $client->getLocation(0));
 
 Passing a `$function` not defined in the WSDL file will throw a `SoapFault`. 
 
+#### withRequestTarget($requestTarget)
+
+This method allows you to change the destination of your SOAP calls. It returns a new Client instance
+with the specified request-target.
```

Comment:
How does this relate to the `getLocation()` method (and its name)?

This could also use some tests I suppose :-)

Reply:
I can rename to setLocation, but thought this may be confusing because getLocation and setLocation are not really getters / setter for the same thing.

This PR adds tests in tests/FunctionalTest.php.